compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -745,7 +747,14 @@ impl Visitor<'tcx> for Validator<'mir, 'tcx> {
                 let (mut callee, substs) = match *fn_ty.kind() {
                     ty::FnDef(def_id, substs) => (def_id, substs),
 
-                    ty::FnPtr(_) => {
+                    ty::FnPtr(fn_sig) => {
+                        // At this point, we are calling a function by raw pointer because
+                        // we know that it is const
+                        if self.ccx.tcx.features().const_fn_pointer
```

Comment:
What do you mean with "by raw pointer"? A raw pointer would be `*const T`/`*mut T`, and I do not think there is such a pointer here.